packages/nixos/gpu.nix

```
@@ -123,8 +123,10 @@ in
     # instead.
 
     # Configure the persistenced for use with CC GPUs (e.g. H100).
-    # TODO(msanft): This needs to be adjusted for non-CC-GPUs.
     # See: https://docs.nvidia.com/cc-deployment-guide-snp.pdf (Page 23 & 24)
+    # Note that the current setup does not work with non-CC-GPUs. (For most use-cases)
```

```suggestion
    # Note that the current setup does not support all use cases with non-CC-GPUs.
```

The point is that not all APIs are working and thus the generic use case is unsupported, right?